git_machete/cli.py

```diff
@@ -556,6 +561,16 @@ def launch(orig_args: List[str]) -> None:
                     machete_client.annotate(branch, parsed_cli.annotation_text)
                 else:
                     machete_client.print_annotation(branch)
+        elif cmd == "clean":
+            machete_client.read_definition_file()
+            machete_client.delete_untracked(opt_yes=cli_opts.opt_yes,
+                                            opt_merge=cli_opts.opt_merge,
+                                            opt_no_interactive_rebase=cli_opts.opt_no_interactive_rebase,
+                                            opt_no_edit_merge=cli_opts.opt_no_edit_merge)
```

Comment:
Hmmm none of these options is going to be passed anyway right? 🤔 should these even be configurable?

Reply:
Well, you are right, they are not passed at the moment but they could be, since `clean` does merge / rebase if necessary, should i add it?

Reply:
> should i add it?

Whoops no, let's not extend the CLI unless necessary... I'd rather remove the params from the method instead!